src/previewFrame/index.js

```diff
@@ -7,57 +7,16 @@ import uniq from 'lodash/uniq';
 import loopBreaker from 'loop-breaker';
 import libraries from '../config/libraries';
 import previewFrameLibraries from '../config/previewFrameLibraries';
+import highlighterCss from '../../templates/highlighterCss.txt';
+import errorHandlerScript from './errorHandlerScript';
+import elementHighlighterScript from './elementHighlighterScript';
+import alertAndPromptReplacementScript
+  from './alertAndPromptReplacementScript';
```

Comment:
I really like this refactor—particularly that the modules are exported as strings of source code. Would make me very nervous otherwise. But a very nice 80/20 vs. the more radical proposal I had made initially.